sbp_register.py

```diff
@@ -172,4 +200,33 @@ def insert(self, register):
         if (sel is None) or len(sel) != 1:
             return
 
-        self.view.window().run_command("sbp_register_do_insert", {"content": sbp_registers.get(register)})
+        self.view.window().run_command("sbp_register_do_insert", {"content": sbp_text_registers.get(register)[3]})
+
+class SbpChooseAndYankRegister(jove.SbpTextCommand):
+    
+    def run_cmd(self, util):
+        # items is an array of (index, text) pairs
+        items = sbp_text_registers.get_text_registers()
+
+        def on_done(idx):
+            if idx >= 0:
+                util.run_command("sbp_register_do_insert", {"content": sbp_text_registers.get(items[idx][0])})
+ 
+        if items:
+            sublime.active_window().show_quick_panel([item[0] + ": " + item[1] for item in items], on_done)
+        else:
+            sublime.status_message('Nothing in history')
+class SbpChooseAndYankPoint(jove.SbpTextCommand):
+    
+    def run_cmd(self, util):
+        # items is an array of (index, text) pairs
+        items = sbp_point_registers.get_point_registers()
+
+        def on_done(idx):
+            if idx >= 0:
+                util.run_command("sbp_point_from_register", {"register": items[idx][0]})
```

Comment:
This doesn't work right because I'm not sure how to call commands in Sublime properly. I tried to pass in the register so it will then bypass the panel coming up to ask the user and just run the `insert` function which I know works. This method doesn't work so any suggestion what to do on this. Or do I have to declare it as `jove.SbpTextCommand` to get it working?